### DISCUSSION TITLE
Lab 1

### DISCUSSION THREAD

**Discussion by giswqs** [giswqs/geog-414#3] [Q&A]
The map in Lab 1 looks cool. I want to use my own data. Can you show me how to do that?

**Comment by giswqs** [comment]
What data do you have? Share the data.

**Comment by giswqs** [comment]
how about this example:

```python
import leafmap
m = leafmap.Map()
m
```

![image](https://github.com/giswqs/geog-414/assets/5016453/fde717bd-0d44-4aed-ab61-f596d3c1e410)

**Comment by ENwagbara97** [comment]
Good Day, Prof. My name is Ebubechukwu Nwagbara. I got the opportunity to join your course through the United Nations University platform. I was working on the Lab 1 assignment and I noticed that this url "https://open.gishub.org/data/vector/cables.geojson" is broken, and hence causing an error, please sir i kindly request for a Url so that i can submit the assignment or maybe tell me how to go about it. Thank you Sir.
